src/App/SettlementWindows/actions.ts

```diff
@@ -20,8 +20,24 @@ import {
   SettlementWindow,
   DateRanges,
   FilterNameValue,
+  SettlementModel,
+  SET_SETTLEMENT_MODELS,
+  REQUEST_SETTLEMENT_MODELS,
+  SET_SELECTED_SETTLEMENT_MODEL,
 } from './types';
 
+// Functions that create actions to update the state.
+// We use @reduxjs/toolkit to save use some boilerplate code.
+// const increment = createAction<number | undefined>('counter/increment')
+
+// let action = increment()
+// { type: 'counter/increment' }
+
```

Comment:
remove the commented code if not required

Reply:
As someone thats been on/off with the reporting ui, I find these refreshers in react/redux very helpful.